Add improvements to The Pit, Fix missing light layer for guns

Fueredoriku opened the pull request:
- Add new walls that obscure players from enemy spawnpoints from their own
- Add new props and decals to create more memorable areas for navigation
- Add triplanar projection toon material and subgraph
- Fix terrain and lighting in the pit so the map looks more like crater town
- Fix correct lightlayers for multiple augments and objects for correct decals
- Fix AIs getting stuck at the bottom of the pit by adding an AI only deathbox
- Fix colliders of chips and ammo boxed being slightly too small

toberge commented on Assets/Prefabs/Interactables/CollectableAmmo.prefab:
```diff
@@ -182,8 +183,8 @@ BoxCollider:
-  m_Size: {x: 0.7, y: 0.95608616, z: 0.7}
-  m_Center: {x: 0, y: 0.7280432, z: 0}
+  m_Size: {x: 2, y: 2, z: 2}
+  m_Center: {x: 0, y: 1, z: 0}
```
hva skjedd her? du scala opp collideren? isåfall støtter jeg det veldig :+1:

Fueredoriku replied:
Jepp! Har fått så mye feedback om det, så tenkte å gjøre det mens jeg husket, men så glemte jeg å skrive det ned <.<

toberge commented on Assets/Prefabs/Interactables/CollectableChip.prefab:
```diff
@@ -180,7 +180,7 @@ BoxCollider:
   m_Enabled: 1
   serializedVersion: 3
-  m_Size: {x: 1, y: 1, z: 1}
+  m_Size: {x: 2, y: 2, z: 2}
```
samme her right?

Fueredoriku replied:
Jupp

toberge commented on Assets/Scripts/Interactables/DestroyOnDeath.cs:
```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DestroyOnDeath : MonoBehaviour
```
hvor er dette brukt hen?

Fueredoriku replied:
CBT-skiltene, destroyer hingejointen

toberge reviewed (approved):
overall very good, but we should playtest and possibly alter the wall obstacles

Fueredoriku committed:
Add improvements to The Pit, fix missing light layer for guns

- Add new walls that obscure players from enemy spawnpoints from their own
- Add new props and decals to create more memorable areas for navigation
- Fix terrain and lighting in the pit so the map looks more like crater town
- Fix correct lightlayers for multiple augments and objects for correct decals
- Fix AIs getting stuck at the bottom of the pit by adding an AI only deathbox
